Dockerfile

```diff
@@ -50,12 +50,11 @@ RUN apk --no-cache add --virtual \
 
 WORKDIR /app
 
-COPY --from=build /app/node_modules /app/node_modules
-COPY --from=build /app/.next /app/.next
 COPY --from=build /app/next.config.js /app/next.config.js
 COPY --from=build /app/public /app/public
 COPY --from=build /app/package.json /app/package.json
-COPY --from=build /app/yarn.lock /app/yarn.lock
+COPY --from=build /app/.next/standalone /app
```

Comment:
Not sure if this was intentional, but I believe this is going to wipe the destination `/app` dir and replace it with the contents of the source `/app/.next/standalone` dir.
Did you mean `COPY --from=build /app/.next/standalone /app/.next/standalone` instead?

Reply:
the `standalone` directory is the output of this new feature.  one can run the application directly from the `/app/.next/standalone` directory, or copy it out into another location and run it from there.

to your point, it does not wipe the destination, it only affects duplicate files and it looks like the line #57 redundant so I will remove that.